### PR TITLE
Update links in footer

### PR DESCRIPTION
**Pull request by ArseniiPetrovich**
Add four new networks:
Aerum
Callisto
Lukso
RSK
@ayrat555 @igorbarinov, please, review.

### CODE REVIEW

**Comment by ArseniiPetrovich** [comment]
Does Lukso is a prod or test network?

**Comment by igorbarinov** [comment]
> Does Lukso is a prod or test network?

Lukso L14 testnet

**Comment by ArseniiPetrovich** [comment]
> > Does Lukso is a prod or test network?
> 
> Lukso L14 Testnet

Thanks, updated the PR

File: apps/block_scout_web/config/config.exs [hunk 1]
```diff
@@ -68,6 +68,23 @@ config :block_scout_web,
     %{
       title: "Ethereum Classic",
       url: "https://blockscout.com/etc/mainnet"
+    },
+    %{
+      title: "Lukso's Network",
```
Comment (vbaranov) [inline apps/block_scout_web/config/config.exs]: maybe, to rename to `LUKSO Network` as in the official repo https://github.com/lukso-network/lukso-chain-spec#configuration-file-for-lukso-network-networks?

**Comment by vbaranov** [comment]
@ArseniiPetrovich to fix `check_formatted` test, please, execute `mix format` from the `./apps/block_scout_web`

**Comment by vbaranov** [comment]
Should be merged after instances will be ready.

**Pull request merged**